### DISCUSSION TITLE
Text to speech API?

### DISCUSSION THREAD

**Discussion by ks233ever** [deepgram/community#626] [General help] [answered]
Does Deepgram have a text to speech API? The only thing I saw was a real time text to speech API waitlist called Aura

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @ks233ever, yes, Aura is our text-to-speech API that has been in a private beta for the past couple of months. We've been making rapid improvements to it based on this early feedback. However, we're on the cusp of opening it up for wider use! Keep an eye out on our docs/website. I'll also aim to share the link for you here soon once it's available.

**Comment by natalierutgers** [comment, marked as answer]
Hi @ks233ever, Aura just entered into Early Access today! You can get started with your existing Deepgram API key and our [getting started guide](https://developers.deepgram.com/docs/text-to-speech). Let us know if you have any questions.

**Reply by ks233ever** [reply]
awesome thanks!
